scripts/chain_configs/localterra.json

```diff
@@ -1,54 +1,25 @@
 {
   "generalInfo": {
-    "multisig": "terra17lmam6zguazs5q5u6z5mmx76uj63gldnse2pdp",
-    "astro_token": "terra1wn625s4jcmvk0szpl85rj5azkfc6suyvf75q6vrddscjdphtve8stalnth",
-    "xastro_token": "terra1g4xlpqy29m50j5y69reguae328tc9y83l4299pf2wmjn0xczq5jsnem6vt",
-    "factory_addr": "terra1xx3aupmgv3ce537c0yce8zzd3sz567syaltr2tdehu3y803yz6gsyz99er",
-    "generator_addr": "terra123gcwlskafnxw4lvkjeacmd5tqzl27gd8xxxnrghe2qhdxtluskqhjcwgd"
+    "multisig": "terra1x46rqay4d3cssq8gxxvqz8xt6nwlz4td20k38v",
+    "astro_token": "",
+    "xastro_token": "",
+    "factory_addr": "",
+    "generator_addr": ""
   },
   "teamUnlock": {
     "admin": null,
     "initMsg": {
       "owner": null,
       "astro_token": null,
-      "max_allocations_amount": "2000000000"
+      "max_allocations_amount": "0"
     },
     "label": "Astroport Builder Unlocking Contract",
     "change_owner": false,
     "propose_new_owner": {
-      "owner": "terra17lmam6zguazs5q5u6z5mmx76uj63gldnse2pdp",
+      "owner": "",
       "expires_in": 604800
     },
-    "setup_allocations": {
```

Comment:
Why you remove allocations for Localterra?

Reply:
Because we don't have these addresses upon a new LocalTerra instance is created

Reply:
We need to have at least one allocation example

Reply:
Added